pubspec.yaml

```diff
@@ -1,16 +1,16 @@
 name: cached_network_image
 description: Flutter library to load and cache network images. Can also be used with placeholder and error widgets.
-version: 0.4.1+1
-author: Rene Floor <pub@renefloor.nl>
+version: 0.4.2
+authors:
+  - Rene Floor <pub@renefloor.nl>
+  - Charles Crete <charles@cretezy.com>
 homepage: https://github.com/renefloor/flutter_cached_network_image
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.0.0-dev <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  flutter_cache_manager: "^0.1.1"
-
-dev_dependencies:
-  test: ^0.12.0
\ No newline at end of file
+  flutter_cache_manager:
+    git: https://github.com/Cretezy/flutter_cache_manager.git
```

Comment:
I feel like this would not be acceptable for a public plugin.

Reply:
This is meant until the PR for that plugin is merged, it says in the description that it's a temporary dependency

Reply:
Thanks for the Pull Request. I was using both the dev and beta, but failed.  I'll be using your fork until the author updates this package.